pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -1038,32 +1040,34 @@ private void setupJobs(@NonNull Config cfg, @NonNull KitUtil kitUtil, @NonNull N
 
                 createScheduleJob(scheduler, eventUtil, notificationUtil, DDPEventJob.class, "TRIGGER_DDP_EVENT",
                         cfg.getString(ApplicationConfigConstants.QUARTZ_CRON_EXPRESSION_FOR_DDP_EVENT_TRIGGER),
-                        new DDPEventTriggerListener(), null);
+                        new DDPEventTriggerListener());
 
                 // currently not needed anymore but might come back
                 // createScheduleJob(scheduler, eventUtil, notificationUtil, ExternalShipperJob.class, "CHECK_EXTERNAL_SHIPPER",
                 // cfg.getString(ApplicationConfigConstants.QUARTZ_CRON_EXPRESSION_FOR_EXTERNAL_SHIPPER),
                 // new ExternalShipperTriggerListener(), cfg);
 
                 createScheduleJob(scheduler, null, null, EasypostShipmentStatusJob.class, "CHECK_STATUS_SHIPMENT",
-                        cfg.getString(ApplicationConfigConstants.QUARTZ_CRON_STATUS_SHIPMENT), new EasypostShipmentStatusTriggerListener(),
-                        cfg);
+                        cfg.getString(ApplicationConfigConstants.QUARTZ_CRON_STATUS_SHIPMENT), new EasypostShipmentStatusTriggerListener());
 
-
-                logger.info("Setup Job Scheduler...");
-                try {
-                    scheduler.start();
-                    logger.info("Job Scheduler setup complete.");
-                } catch (Exception ex) {
-                    throw new RuntimeException("Unable to setup Job Scheduler.", ex);
-                }
-            } catch (SchedulerException e) {
-                throw new RuntimeException("Could not create scheduler ", e);
+                startScheduler(scheduler);
+            } catch (Exception e) {
+                throw new DsmInternalError("Could not create scheduler ", e);
             }
         }
         setupErrorNotifications(cfg, schedulerName);
     }
 
+    private void startScheduler(Scheduler scheduler) throws DsmInternalError {
+        logger.info("Setup Job Scheduler...");
+        try {
+            scheduler.start();
+            logger.info("Job Scheduler setup complete.");
+        } catch (Exception ex) {
+            throw new DsmInternalError("Unable to setup Job Scheduler.", ex);
```

Comment:
It looks like in all cases this throw will be caught (above) and converted to a slightly different exception. 
Does it make sense to just eliminate this catch? It's it unclear what value it provides.